code/game/objects/items/tools/misc_tools.dm

```diff
@@ -65,6 +52,12 @@
 		to_chat(user, "<span class='notice'>You turn off \the [src].</span>")
 
 
+/obj/item/tool/hand_labeler/attackby(obj/item/I, mob/user, params)
+	..()
```

Comment:
`. = ..()`

Reply:
Is there any difference between the two?

Reply:
It assigns the value of the parent call to `.`